src/routes/namespaces/[namespace]/workflows/_workflows-summary-table.svelte

```diff
@@ -10,3 +10,9 @@
   </div>
   <slot />
 </section>
+
+<style lang="postcss">
+  section {
+    height: calc(100vh - 20%);
```

Comment:
Is this different than just saying `80vh`? Like, this should work, but only because it's guessing at the other elements on the page, right?